.github/workflows/package-linux.yml

```diff
@@ -81,3 +81,66 @@ jobs:
+    strategy:
+      fail-fast: false
+      matrix:
+        vec: [
+          # Ubuntu 24.04
+          { friendlyName: "Ubuntu 24.04 x64", config: "Release", os: "ubuntu-24.04", arch: "x64",   tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-24.04-helix-amd64", xdp: "-UseXdp", dotnetVersion: "9.0" },
+          { friendlyName: "Ubuntu 24.04 ARM32", config: "Release", os: "ubuntu-24.04", arch: "arm",   tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-24.04-helix-arm32v7", dotnetVersion: "9.0" },
+          { friendlyName: "Ubuntu 24.04 ARM64", config: "Release", os: "ubuntu-24.04", arch: "arm64", tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-24.04-helix-arm64v8", dotnetVersion: "9.0" },
+          # Ubuntu 22.04
+          { friendlyName: "Ubuntu 22.04 x64", config: "Release", os: "ubuntu-22.04", arch: "x64",   tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-amd64", dotnetVersion: "9.0" },
+          { friendlyName: "Ubuntu 22.04 ARM64", config: "Release", os: "ubuntu-22.04", arch: "arm64", tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-arm64v8", dotnetVersion: "9.0" },
+          # Ubuntu 20.04
+          { friendlyName: "Ubuntu 20.04 x64", config: "Release", os: "ubuntu-20.04", arch: "x64",   tls: "openssl", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-helix-amd64", dotnetVersion: "9.0" },
+          { friendlyName: "Ubuntu 20.04 ARM64", config: "Release", os: "ubuntu-20.04", arch: "arm64", tls: "openssl", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-helix-arm64v8", dotnetVersion: "9.0" },
+          # Debian 12
+          { friendlyName: "Debian 12 x64", config: "Release", os: "ubuntu-22.04", arch: "x64",   tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-helix-amd64", dotnetVersion: "9.0" },
+          { friendlyName: "Debian 12 ARM32", config: "Release", os: "ubuntu-22.04", arch: "arm",   tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-helix-arm32v7", dotnetVersion: "9.0" },
+          { friendlyName: "Debian 12 ARM64", config: "Release", os: "ubuntu-22.04", arch: "arm64", tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-helix-arm64v8", dotnetVersion: "9.0" },
+          # CBL-Mariner 2.0
+          # { config: "Release", os: "ubuntu-24.04", arch: "x64",   tls: "openssl", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-helix-amd64" },
+          # Azure Linux 3.0
+          # { config: "Release", os: "ubuntu-24.04", arch: "arm64", tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-helix-arm64v8" },
+          # { config: "Release", os: "ubuntu-24.04", arch: "x64",   tls: "openssl3", image: "mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-helix-amd64", xdp: "-UseXdp" },
```

Comment:
Why disabled?

Reply:
Those images don't have sudo in their installed packages, so we can't modify them but I'll replace them with another image for same distro.